docs/Salesforce-batchsource.md

```diff
@@ -18,10 +18,10 @@ Configuration
 
 **Password:** Salesforce password.
 
-**Client Id:** Application Client Id. This is also called "Consumer Key" in Salesforce UI.
-To create Client Id user needs to create a connected application in Salesforce first.
+**Consumer Key:** Application Consumer Key. This is also known as "Client Id".
+To create Consumer Key user needs to create a connected application in Salesforce first.
```

Comment:
we should not be calling the user "user" in the documentation.

A Salesforce connected application must be created in order to get a consumer key.

Reply:
done